aws-android-sdk-pinpoint-test/src/androidTest/java/com/amazonaws/mobileconnectors/pinpoint/analytics/EndpointProfileIntegrationTest.java

```diff
@@ -81,9 +81,8 @@ public void setUp() throws Exception {
                 appContext,
                 getPackageConfigure("pinpoint")
                         .getString("identity_pool_id"),
-                Regions.fromName(getPackageConfigure("pinpoint")
-                        .getString("identity_pool_id_region")));
-        pinpointConfiguration = new PinpointConfiguration(appContext,
+                regions);
+                pinpointConfiguration = new PinpointConfiguration(appContext,
```

Comment:
Indentation looks off, should it be -4? Maybe this is a `\t` char being rendered as a double indent (2 x 4 spaces)?

Reply:
😞 Oops.